sdk/eventhub/azure-eventhubs/azure/eventhub/client_abstract.py

```diff
@@ -196,45 +162,37 @@ def _create_properties(self, user_agent=None):  # pylint: disable=no-self-use
         :rtype: dict
         """
         properties = {}
-        properties["product"] = "eventhub.python"
-        properties["version"] = __version__
-        properties["framework"] = "Python {}.{}.{}".format(*sys.version_info[0:3])
-        properties["platform"] = sys.platform
-
-        final_user_agent = 'azsdk-python-eventhub/{} ({}; {})'.format(
-            __version__, properties["framework"], sys.platform)
+        product = "azure-eventhub"
```

Comment:
product = "azsdk-python-eventhubs"

Reply:
I confirmed with PM. We should use azure-eventhub, the same as pypi package name.